src/FileSystem/FilesFinder.php

```diff
@@ -68,7 +68,10 @@ private function findInDirectories(array $directories, array $suffixes): array
 
         $filePaths = [];
         foreach ($finder as $fileInfo) {
-            $filePaths[] = $fileInfo->getRealPath();
+            $path = $fileInfo->getRealPath();
```

Comment:
`@var` should can be used to handle PHPStan notice:

```suggestion
            /** @var string|false $path */
            $path = $fileInfo->getRealPath();
```